src/org/traccar/Context.java

```diff
@@ -413,9 +423,17 @@ public static void init(String[] arguments) throws Exception {
 
     }
 
+    private static EventForwarder getConfiguredEventForwarder() {
+        if (Context.getConfig().getBoolean("event.forward.payloadAsParamMode")) {
+            return new FormTypeEventForwarder();
+        }
+        return new JsonTypeEventForwarder();
+    }
+
     public static void init(IdentityManager testIdentityManager) {
         config = new Config();
         objectMapper = new ObjectMapper();
+        objectWriterPrettyPrinter = objectMapper.writerWithDefaultPrettyPrinter();
```

Comment:
I don't think we need both options at the same time.

Reply:
This was already explained:

- I like the idea to keep 2 separate serializers, one for internal use and another one eventually for external communication, new modules could decide to use one or another.

- I prefered this approach because they are 2 different objects, _ObjectWriter_ and  _ObjectMapper_, see EventForwarder.java at line 115

- I did not found a method that could let you to keep an instance of ObjectMapper having a default writer with prettyPrinter.

- The main purpose of having a specific instance of the writer is because .writerWithDefaultPrettyPrinter() will leak a new writer instance each time that is called so i preferred to do it once and keep it.

Reply:
I don't see any good reason to have two. The only reason I can think of to use "pretty" writer is some kind of debugging. In that case we can enable it globally.

Reply:
Done